src/components/Image/index.js

```diff
@@ -27,12 +27,16 @@ class Image extends React.Component {
     getImageSource() {
         const source = this.props.source;
         let imageSource = source;
+
         if (this.props.isAuthTokenRequired) {
             // There is currently a `react-native-web` bug preventing the authToken being passed
             // in the headers of the image request so the authToken is added as a query param.
             // On native the authToken IS passed in the image request headers
             const authToken = lodashGet(this.props, 'session.encryptedAuthToken', null);
-            imageSource = {uri: `${source.uri}?encryptedAuthToken=${encodeURIComponent(authToken)}`};
+
+            const url = new URL(source.uri);
+            url.searchParams.append('encryptedAuthToken', encodeURIComponent(authToken));
```

Comment:
```suggestion
            url.searchParams.append('encryptedAuthToken', authToken);
```

We don't need to encode the auth token.